ui/shared/components/form/Inputs.jsx

```diff
@@ -68,6 +55,7 @@ export const IntegerInput = React.memo(({ onChange, min, max, value, ...props })
     min={min}
     max={max}
     onChange={setIntVal(onChange, min, max)}
+    onClick={stopPropagation}
```

Comment:
why is this changing? You don't seem to use `IntegerInput` anywhere in this PR?

Reply:
Sorry that was when I removed stopPropagation from the VariantSearchFormPanels header.
See above: https://github.com/broadinstitute/seqr/pull/2790#discussion_r919757099

The other inputs used in headers naturally stop propogation, but this one didn't, so it needed it to be added or the accordian would open/close whenever the input was clicked.

I have reverted it now 👍